mautrix_telegram/example-config.yaml

```
@@ -222,6 +222,11 @@ bridge:
     image_as_file_size: 10
     # Maximum number of pixels in an image before sending to Telegram as a document. Defaults to 4096x4096 = 16777216.
     image_as_file_pixels: 16777216
+    # Maximum size of Telegram documents before linking to Telegrm instead of bridge
+    # to Matrix media.
+    document_as_link_size:
+        channel: 10
+        bot: 10
```

These should default to null/unlimited

```suggestion
        channel:
        bot:
```